ui/packages/consul-ui/app/components/consul/auth-method/view/index.hbs

```diff
@@ -0,0 +1,217 @@
+  <div class="consul-auth-method-view">
+  {{#if (eq @item.Type 'kubernetes')}}
+    <dl>
+      <dt>Type</dt>
+      <dd><Consul::AuthMethod::Type @item={{@item}} /></dd>
+
+    {{#each (array "MaxTokenTTL" "TokenLocality" "DisplayName" "Description") as |value|}}
+      {{#if (get @item value)}}
+      <dt>{{t (concat "components.consul.auth-method.definition-list." value)}}</dt>
+      <dd>{{get @item value}}</dd>
+      {{/if}}
+    {{/each}}
+      {{#if @item.Config.Host}}
+      <dt>Host</dt>
+      <dd>
+        <CopyButton @value={{@item.Config.Host}} @name="Host"/>
+        <span>{{@item.Config.Host}}</span>
+      </dd>
+      {{/if}}
+      {{#if @item.Config.CACert}}
+      <dt>CA Cert</dt>
+      <dd>
+        <Certificate @item={{@item.Config.CACert}} @name={{t 'models.auth-method.Config.CACert'}} />
+      </dd>
+      {{/if}}
+      {{#if @item.Config.ServiceAccountJWT}}
+      <dt>Service account JSON Web Token</dt>
+      <dd>
+        <CopyButton @value={{@item.Config.ServiceAccountJWT}} @name="Service account JSON Web Token" />
+        <span>
```

Comment:
I think there're some `t`s missed here, not sure if you meant to do all these or not? Seems like sometimes `t` is used and sometimes not?

Reply:
Whoops! Looks like I forgot to fix up the 'kubernetes' section.